### PR TITLE
[#157730702] Use npm audit

### PR DESCRIPTION
**Pull request by chrisfarms**
## What

Swap out snyk for npm audit for checking for issues with dependencies during `npm test` runs.

## How to review

* Check that the new `npm run test:audit` returns a full audit
* Check that the `npm run test:vulnerabilities` looks like it will return a non-zero exit code when a high or critical issue is found

## Who can review

Not @chrisfarms

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,7 +73,7 @@
     "tslint-microsoft-contrib": "5.0.3",
     "typescript": "2.8.1",
     "url-loader": "1.0.1",
-    "webpack": "4.4.1",
+    "webpack": "^4.10.2",
```
Comment (paroxp) [inline package.json]: Can we pin the versions? :D

**Comment by paroxp** [comment]
Worth to mention, `npm install` in order.

File: package.json (hunk 1 above)
Comment (chrisfarms) [reply package.json]: well the lock file kinda does this already ... but ok, one sec :)

Comment (chrisfarms) [reply package.json]: done ... I intentionally left the `npm` only pinned to ^6.

**Pull request merged**